e2e/star-search.spec.ts

```diff
@@ -139,19 +141,45 @@ test("StarSearch shared prompt (Logged Out Flow)", async ({ page }) => {
   // open share prompt options menu
   await sharePopupMenuTrigger.click();
 
-  await expect(page.getByRole("menuitem", { name: "Share to Twitter/X", exact: true })).toBeVisible();
-  await expect(page.getByRole("menuitem", { name: "Share to LinkedIn", exact: true })).toBeVisible();
-  await expect(page.getByRole("menuitem", { name: "Copy link", exact: true })).toBeVisible();
+  const shareToTwitterMenuItem = page.getByRole("menuitem", { name: "Share to Twitter/X", exact: true });
+  const shareToLinkedInMenuItem = page.getByRole("menuitem", { name: "Share to LinkedIn", exact: true });
+  const copyLinkMenuItem = page.getByRole("menuitem", { name: "Copy link", exact: true });
+
+  await expect(shareToTwitterMenuItem).toBeVisible();
+  await expect(shareToTwitterMenuItem.locator("a")).toHaveAttribute(
+    "href",
+    `https://twitter.com/intent/tweet?text=Here%27s+my+StarSearch+prompt%21%0A%0ATry+it+out+for+yourself.+%23StarSearch&url=${encodeURIComponent(
+      BASE_URL
+    )}%2Fstar-search%3Fprompt%3DWho%2Bare%2Bthe%2Bmost%2Bprevalent%2Bcontributors%2Bto%2Bthe%2BTypeScript%2Becosystem%253F`
+  );
+
+  await expect(shareToLinkedInMenuItem).toBeVisible();
+  await expect(shareToLinkedInMenuItem.locator("a")).toHaveAttribute(
+    "href",
+    `https://www.linkedin.com/sharing/share-offsite/?url=${encodeURIComponent(
+      BASE_URL
+    )}%2Fstar-search%3Fprompt%3DWho%2Bare%2Bthe%2Bmost%2Bprevalent%2Bcontributors%2Bto%2Bthe%2BTypeScript%2Becosystem%253F`
+  );
+  await expect(copyLinkMenuItem).toBeVisible();
+
+  if (["webkit", "Desktop Safari", "Mobile Safari"].includes(browserName)) {
```

Comment:
Added a test to ensure copying a link to the clipboard works. Currently not supported by webkit/Safari. See https://github.com/microsoft/playwright/issues/13037